### ISSUE TITLE
Test after installation failed

### ISSUE THREAD

**Issue by rckeerthivasan** [giesselmann/STRique#27]
Hi, I just installed the tool STRique and I ran the test command using 'python3 scripts/STRique_test.py' and I received the following errors. Thank you. 

`EEEE
======================================================================
ERROR: test_Detection (__main__.DetectionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "scripts/STRique_test.py", line 55, in test_Detection
    dt.add_target('c9orf72', repeat, prefix, suffix)
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 566, in add_target
    flankedRepeatHMM(repeat, prefix, suffix, self.pm, self.HMM_config),
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 402, in __init__
    self.__build_model__()
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 431, in __build_model__
    self.bake(merge='All')
  File "pomegranate/hmm.pyx", line 795, in pomegranate.hmm.HiddenMarkovModel.bake
  File "/home/ken/Kens/softwares.3/STRique/v1/lib64/python3.8/site-packages/networkx/classes/reportviews.py", line 718, in <genexpr>
    for nbr, dd in nbrs.items()
RuntimeError: dictionary keys changed during iteration

======================================================================
ERROR: test_Interpolation (__main__.DetectionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "scripts/STRique_test.py", line 75, in test_Interpolation
    dt.add_target('fmr1', repeat, prefix, suffix)
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 566, in add_target
    flankedRepeatHMM(repeat, prefix, suffix, self.pm, self.HMM_config),
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 402, in __init__
    self.__build_model__()
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 431, in __build_model__
    self.bake(merge='All')
  File "pomegranate/hmm.pyx", line 795, in pomegranate.hmm.HiddenMarkovModel.bake
  File "/home/ken/Kens/softwares.3/STRique/v1/lib64/python3.8/site-packages/networkx/classes/reportviews.py", line 718, in <genexpr>
    for nbr, dd in nbrs.items()
RuntimeError: dictionary keys changed during iteration

======================================================================
ERROR: test_Modification (__main__.DetectionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "scripts/STRique_test.py", line 114, in test_Modification
    dt.add_target('c9orf72', repeat, prefix, suffix)
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 566, in add_target
    flankedRepeatHMM(repeat, prefix, suffix, self.pm, self.HMM_config),
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 402, in __init__
    self.__build_model__()
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 431, in __build_model__
    self.bake(merge='All')
  File "pomegranate/hmm.pyx", line 795, in pomegranate.hmm.HiddenMarkovModel.bake
  File "/home/ken/Kens/softwares.3/STRique/v1/lib64/python3.8/site-packages/networkx/classes/reportviews.py", line 718, in <genexpr>
    for nbr, dd in nbrs.items()
RuntimeError: dictionary keys changed during iteration

======================================================================
ERROR: test_Normalization (__main__.DetectionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "scripts/STRique_test.py", line 93, in test_Normalization
    dt.add_target('c9orf72', repeat, prefix, suffix)
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 566, in add_target
    flankedRepeatHMM(repeat, prefix, suffix, self.pm, self.HMM_config),
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 402, in __init__
    self.__build_model__()
  File "/home/ken/Kens/softwares.3/STRique/STRique/scripts/STRique.py", line 431, in __build_model__
    self.bake(merge='All')
  File "pomegranate/hmm.pyx", line 795, in pomegranate.hmm.HiddenMarkovModel.bake
  File "/home/ken/Kens/softwares.3/STRique/v1/lib64/python3.8/site-packages/networkx/classes/reportviews.py", line 718, in <genexpr>
    for nbr, dd in nbrs.items()
RuntimeError: dictionary keys changed during iteration

----------------------------------------------------------------------
Ran 4 tests in 0.239s

FAILED (errors=4)
`

**Comment by giesselmann** [comment]
Hi,
looks like some version mismatch for me.
Unfortunately are the versions of pomegranate and networkx crucial.
Can you check that they meet the ones stated in requirements.txt or post the output of pip freeze here?
I haven't tested STRique in py3.8 so far.
Pay

**Comment by rckeerthivasan** [comment]
Hi, 

Thank you for pointing it out. I reorganized the versions installed and it worked. 

Ken. 

Test GGCCCC repeat
Test repeat length: 100
Test repeat length: 200
Test repeat length: 300
.Test GCG repeat
Test repeat length: 100
Test repeat length: 200
Test repeat length: 300
.Test GGCCCC repeat
Test repeat length: 100
Test repeat length: 200
Test repeat length: 300
.Test normalization on short prefix/suffix sequences
Test repeat length: 10
Test repeat length: 20
Test repeat length: 30
Test repeat length: 40
Test repeat length: 50
Test repeat length: 60
Test repeat length: 70
Test repeat length: 80
Test repeat length: 90
.
----------------------------------------------------------------------
Ran 4 tests in 22.581s

OK
